xcodebuild.log

```diff
@@ -0,0 +1,342 @@
+Command line invocation:
```

Comment:
You probably don't want to add this? Maybe to .gitignore?

Reply:
Removed.